src/commands/math/commands.js

```diff
@@ -246,7 +246,7 @@ var SupSub = P(MathCommand, function(_, super_) {
   _.latex = function() {
     function latex(prefix, block) {
       var l = block && block.latex();
-      return block ? prefix + (l.length === 1 ? l : '{' + (l || ' ') + '}') : '';
+      return block ? prefix + ('{' + (l || ' ') + '}') : '';
```

Comment:
are you sure your change doesn't have unwanted side effects? If authors are now inserting logs through mathquill, and they are displayed somewhere with mathjax (e.g. sample solution), then the `{}` will be visible I guess?

Reply:
> are you sure your change doesn't have unwanted side effects?

Yes, using `{...}` as subscript is standard latex and actually, when the subscript content length is `> 1` then its surrounded with curly brackets. I don't exactly know why they treat the `=== 1` case differently, but it actually produces wrong latex.

> If authors are now inserting logs through mathquill, and they are displayed somewhere with mathjax (e.g. sample solution), then the {} will be visible I guess?

No, see here https://staging-amssg.taskbase.com/task/39FEWkqJn8U91O8dHfsZpp/edit.